econml/dml.py

```diff
@@ -50,18 +50,23 @@ class _RLearner(LinearCateEstimator):
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
+
+    inference: string, inference method, or None
+        Method for performing inference.  This estimator supports 'bootstrap'
+        (or an instance of `BootstrapOptions`)
     """
 
     def __init__(self, model_y, model_t, model_final,
```

Comment:
Why do you need to declare at init time whether you are planning to do inference or not?

Reply:
You need to declare it at init time because then we must fit the replicates when the class itself is fit, which (as you note below) may have a large performance penalty.

For some subclasses we'll be able to support different, more performant options.